xclim/core/utils.py

```diff
@@ -684,7 +683,7 @@ def adapt_clix_meta_yaml(  # noqa: C901
     freq_defs = {"annual": "YS", "seasonal": "QS-DEC", "monthly": "MS", "weekly": "W"}
 
     if isinstance(raw, os.PathLike):
-        with open(raw) as f:
+        with open(raw, encoding="utf-8") as f:
```

Comment:
Is there a context where this isn't the default ?

Reply:
Older Windows systems with unique locales can _potentially_ open files with different encodings or in str/bytes mode. This is a suggestion to make these reads bullet-proof.